### DISCUSSION TITLE
Include last-modified field in header

### DISCUSSION THREAD

**Discussion by ghost** [StevenBlack/hosts#1619] [Ideas]
It would be great if StevenBlack lists contain last-modified field in the header. This will allow all subscribers to use this field to verify current version and status. With that field and proper modified date/time in the field subscribers can perform a test before downloading the list. Many other lists eg: orca and urlhause contain the header field.
Please include that in your lists.

**Comment by funilrys** [comment]
Same idea ("subscribers can perform a test"): #1573.

**Comment by ghost** [comment]
I am not sure if this is the same idea. I am not asking to update the actual host file but rather HTTP header information with a standard field to store last date/time the file was changed. Subsribers can use that eg: curl -I to retrive the field value and make a decision to download a complete file or not.
  
​
  
  

  
​Mark Abram
  
***@***.***

**Reply by funilrys** [reply]
Where are you downloading from @markster101? 

If from GitHub, this discussion is out of scope as we can't do anything.
If from a mirror, Steve may respond to this as the principal mirror is managed by him.

**Comment by ghost** [comment]
You are correct. Mirror site does support this feature.
  
​
  
  

  
​Mark Abram
  
***@***.***
